### ISSUE TITLE
Miller Midi

### ISSUE THREAD

**Issue by CasualMainYT** [NoahWantsDie/fnf-shits#41]
A Miller Midi would be sick

**Comment by Plasmoxxie** [comment]
Agreed. The only ones on youtube suck. One of them just isn't accurate and the other one is just an FLP convert without fixing the slides.
